purchase_transport_mode/views/purchase_order_view.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_view_form_inherit" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='currency_id']" position="after">
+                <field name="transport_mode_id" />
+            </xpath>
+            <xpath expr="//sheet" position="before">
+                <field name="transport_mode_status_display" nolabel="1" />
```

Comment:
I'm not sure the position is correct... it would be better before lines IMO and wrapped into an alert maybe.
Also, the field should be protected like this:
```suggestion
                <field name="transport_mode_status_ok" invisible="1" />
                <field name="transport_mode_status_display" nolabel="1"  attrs="{'invisible': [('transport_mode_status_ok', '=', False)]}" />
```